src_new/adapters/prebid.js

```diff
@@ -229,6 +303,17 @@ function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, g
 			bids: [],
 			divID : divID
 		};
+	}else if(CONFIG.getSingleImpressionSetting()){
+		var flag = false;
+		adUnits[code].bids.forEach(function(element) {
```

Comment:
please use meaningful names for variables.
if we are looping on an array of bids then in forEach function it will be meaningful to name the argument as a "bid" than "element".